docs/operator_guide.md

```diff
@@ -123,10 +123,6 @@ In order for this to work DNS entries for the resulting hosts needs to be create
 
 If you wish to expose certain environment variables to every application in your cluster, define them here. Regardless of how a variable is passed in (option, environment variable or in a config file), it must be specified as `<key>=<value>`.
 
-### blacklist / whitelist
-
-This is currently only used for FINN, and will go away soon.
```

Comment:
There are a couple other options (`proxy` and `infrastructure` if memory serves) mentioned in this file that is either no longer used, or needs updated descriptions.

Reply:
- `proxy` will be used for usage tracking if set and usage tracking is enabled
- `infrastructure` is still used to set `FIAAS_INFRASTRUCTURE` in deployed application's environments. I'm not sure if any applications rely on this or not, so I would rather not remove it as part of this PR

Reply:
I've updated the docs for the config parameter to reflect this

Reply:
👍